apps/web/src/pages/Users/UserInformationPage/components/UserCandidatesTable/UserCandidatesTable.tsx

```diff
@@ -0,0 +1,261 @@
+import React from "react";
+import { ColumnDef, createColumnHelper } from "@tanstack/react-table";
+import { useIntl } from "react-intl";
+
+import {
+  Department,
+  FragmentType,
+  PoolCandidate,
+  UserInfoFragment,
+} from "@gc-digital-talent/graphql";
+import {
+  commonMessages,
+  getLanguage,
+  getLocalizedName,
+  getPoolCandidatePriorities,
+  getPoolCandidateStatus,
+} from "@gc-digital-talent/i18n";
+import { notEmpty, unpackMaybes } from "@gc-digital-talent/helpers";
+
+import Table from "~/components/Table/ResponsiveTable/ResponsiveTable";
+import { normalizedText } from "~/components/Table/sortingFns";
+import { getFullNameLabel } from "~/utils/nameUtils";
+import tableMessages from "~/components/PoolCandidatesTable/tableMessages";
+import useRoutes from "~/hooks/useRoutes";
+import { PoolCandidate_BookmarkFragment } from "~/components/CandidateBookmark/CandidateBookmark";
+import { getFullPoolTitleLabel } from "~/utils/poolUtils";
+import processMessages from "~/messages/processMessages";
+import adminMessages from "~/messages/adminMessages";
+import {
+  JobPlacementDialog_Fragment,
+  jobPlacementDialogAccessor,
+} from "~/components/PoolCandidatesTable/JobPlacementDialog";
+import cells from "~/components/Table/cells";
+import accessors from "~/components/Table/accessors";
+
+import {
+  bookmarkCell,
+  bookmarkHeader,
+  candidacyStatusAccessor,
+  candidateNameCell,
+  currentLocationAccessor,
+  finalDecisionCell,
+  notesCell,
+  priorityCell,
+  processCell,
+} from "./helpers";
+
+type PoolCandidateSlice = Pick<
+  PoolCandidate,
+  | "id"
+  | "user"
+  | "notes"
+  | "submittedAt"
+  | "status"
+  | "pool"
+  | "placedDepartment"
+  | "suspendedAt"
+  | "isBookmarked"
+>;
+
+const columnHelper = createColumnHelper<PoolCandidateSlice>();
```

Comment:
Just a question because I was struggling with this but did you try a fragment here?

`const UserCandidatesTableRow_Fragment = graphql()`

Reply:
Am experimenting. Tried something else in https://github.com/GCTC-NTGC/gc-digital-talent/pull/10342/commits/bb60ecac454518dc65f1fe5f5dc7c114ec3699f3

The tricky bit is not losing the typesafe of fragments, as we've seen with passing along into further functions or components you can easily lose the checking. If you pass your data into something with `type: PoolCandidate` you won't know if your fragment was missing something. 

An aside, I am finding it quite difficult to ensure all the RoD fields are present as it is a chain of functions for those and they often have the type PoolCandidate even when only interacting with 2 or 3 fields. Making it so I have to paste a lot of fields into the fragment just in case. So it'll be function A calling B, which calls function C and it passes along the entire pool candidate object and you have to dig around to see what fields it even needed.